src/dashboard/src/pages/ChainCode/ChainCode.js

```diff
@@ -96,136 +89,64 @@ const UploadChainCode = props => {
       destroyOnClose
       title={intl.formatMessage({
         id: 'app.chainCode.form.create.header.title',
-        defaultMessage: 'Upload Chain code',
+        defaultMessage: 'Upload chaincode',
       })}
       confirmLoading={uploading}
-      visible={modalVisible}
+      open={modalVisible}
       onOk={onSubmit}
       onCancel={() => handleModalVisible(false)}
     >
       <Form onFinish={onFinish} form={form} preserve={false}>
         <FormItem
           {...formItemLayout}
           label={intl.formatMessage({
-            id: 'app.chainCode.form.create.name',
-            defaultMessage: 'Name',
+            id: 'app.operator.chainCode.form.create.file',
```

Comment:
Can't find "app.operator.chainCode.form.create.file" in locales.

Reply:
it has been updated in [b2493fd](https://github.com/hyperledger/cello/pull/573/commits/b2493fdc6617449820f656ec7889edfe1dfbd7eb)

Reply:
OK